manifests/common/plumgrid.pp

```diff
@@ -11,6 +14,16 @@
     owner   => root,
     group   => root,
     mode    => '0440',
-    content => 'nova ALL=(root) NOPASSWD: /opt/pg/bin/ifc_ctl_pp *\n',
+    content => "nova ALL=(root) NOPASSWD: /opt/pg/bin/ifc_ctl_pp *\n",
+  }
+
+  openstack::resources::firewall { 'Nova API': port => '8774', }
+  openstack::resources::firewall { 'Nova Metadata': port => '8775', }
+
+  class { '::nova::api':
```

Comment:
The nova::api class is managed in the nova api profile, it does not belong here as part of support for a networking plugin.